tagstudio/resources/translations/en.json

```diff
@@ -54,67 +63,105 @@
     "folders_to_tags.title": "Create Tags From Folders",
     "generic.add": "Add",
     "generic.apply": "Apply",
+    "generic.apply_alt": "&Apply",
     "generic.cancel": "Cancel",
+    "generic.cancel_alt": "&Cancel",
+    "generic.close": "Close",
     "generic.copy": "Copy",
     "generic.cut": "Cut",
+    "generic.save": "Save",
     "generic.delete": "Delete",
+    "generic.delete_alt": "&Delete",
+    "generic.continue": "Continue",
     "generic.done": "Done",
+    "generic.done_alt": "&Done",
     "generic.edit": "Edit",
+    "generic.edit_alt": "&Edit",
+    "generic.rename": "Rename",
+    "generic.rename_alt": "&Rename",
+    "generic.overwrite": "Overwrite",
+    "generic.overwrite_alt": "&Overwrite",
+    "generic.skip": "Skip",
+    "generic.skip_alt": "&Skip",
     "generic.navigation.back": "Back",
     "generic.navigation.next": "Next",
     "generic.paste": "Paste",
     "generic.recent_libraries": "Recent Libraries",
+    "generic.filename": "Filename",
     "help.visit_github": "Visit GitHub Repository",
     "home.search_entries": "Search Entries",
     "home.search_library": "Search Library",
     "home.search_tags": "Search Tags",
     "home.search": "Search",
+    "home.thumbnail_size.extra_large": "Extra Large Thumbnails",
+    "home.thumbnail_size.large": "Large Thumbnails",
+    "home.thumbnail_size.medium": "Medium Thumbnails",
+    "home.thumbnail_size.small": "Small Thumbnails",
+    "home.thumbnail_size.mini": "Mini Thumbnails",
     "home.thumbnail_size": "Thumbnail Size",
-    "ignore_list.add_extension": "Add Extension",
+    "ignore_list.add_extension": "&Add Extension",
     "ignore_list.mode.exclude": "Exclude",
     "ignore_list.mode.include": "Include",
-    "ignore_list.mode.label": "List Mode",
+    "ignore_list.mode.label": "List Mode:",
```

Comment:
Do you think it would be better to keep the colon out of the base translation then add it after the fact for any labels that benefit from it? To me I feel this would allow us to be more flexible with UI changes without needing to update the translations for things like this, but I'd like to get your thoughts.

Reply:
depends on whether colons are universally used in all languages (which I kind of doubt), because if not there would be some value in leaving that to translators. Code wise it is cleaner to leave it to the translation strings, but that's not really important besides maybe being a tie breaker

Reply:
After doing a little bit of research, yeeeeah seems like including a colon inside the string is the right move. Adding it in post could cause all sorts of unintended [extra meanings](https://en.wikipedia.org/wiki/Colon_(punctuation)#Usage_in_other_languages) in other languages, in addition to some languages just straight up not using it.